### PR TITLE
Rename navigation guidance day/night style methods to navigation day/night style methods.

### PR DESCRIPTION
**Pull request by MaximAlien**
Closing #2549.

### CODE REVIEW

File: MapboxNavigation/MGLStyle.swift [hunk 1]
```diff
@@ -25,55 +25,31 @@ extension MGLStyle {
...
-     Returns the URL to the given version of the Mapbox Navigation Preview Day style. Available versions are 1, 2, 3, and 4.
-     
-     We only have one version of Navigation Preview style in China, so if you switch your endpoint to .cn, it will return the default day style.
-     */
-    public class func navigationPreviewDayStyleURL(version: Int) -> URL {
+    public class func navigationDayStyleURL(version: Int) -> URL {
         if MGLAccountManager.hasChinaBaseURL {
             return mapboxChinaDayStyleURL
         }
         return URL(string:"mapbox://styles/mapbox/navigation-guidance-day-v\(version)")!
```
Comment (MaximAlien) [inline MapboxNavigation/MGLStyle.swift:44]: Even though methods were renamed URLs were left intact (still contain `guidance`). If there are updated URLs for versions prior to v5 please let me know. I've done a brief search and do not see anything.

File: MapboxNavigation/MGLStyle.swift [hunk 2]
```diff
@@ -25,55 +25,31 @@ extension MGLStyle {
      */
     public class var navigationNightStyleURL: URL {
         get {
-            if(MGLAccountManager.hasChinaBaseURL){
-                return mapboxChinaDayStyleURL
+            if MGLAccountManager.hasChinaBaseURL {
+                return mapboxChinaNightStyleURL
```
Comment (MaximAlien) [inline MapboxNavigation/MGLStyle.swift:29]: Not sure why `mapboxChinaDayStyleURL` was used in `navigationNightStyleURL`, probably was used by mistake.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,8 @@
...
 * Replaced the `MGLStyle.navigationPreviewDayStyleURL` and `MGLStyle.navigationGuidanceDayStyleURL` properties with `MGLStyle.navigationDayStyleURL`, and replaced `MGLStyle.navigationPreviewNightStyleURL` and `MGLStyle.navigationGuidanceNightStyleURL` with `MGLStyle.navigationNightStyleURL`. ([#2523](https://github.com/mapbox/mapbox-navigation-ios/pull/2523))
+* Replaced the `MGLStyle.navigationGuidanceDayStyleURL(version:)` and `navigationGuidanceNightStyleURL(version:)` methods with `MGLStyle.navigationDayStyleURL(version:)` and `MGLStyle.navigationNightStyleURL(version:)` respectively. ([#2567](https://github.com/mapbox/mapbox-navigation-ios/pull/2567))
+* Removed the `MGLStyle.navigationPreviewDayStyleURL(version:)` and `MGLStyle.navigationPreviewNightStyleURL(version:)` methods. ([#2567](https://github.com/mapbox/mapbox-navigation-ios/pull/2567))
```
Comment (1ec5) [inline CHANGELOG.md:23]: By the way, the blurb for #2523 tries to simplify things by describing it as combining symbols.

File: MapboxNavigation/MGLStyle.swift (hunk 1 above)
Comment (1ec5) [reply MapboxNavigation/MGLStyle.swift:44]: The v5 style probably won‘t have “guidance“ in the URL.

Comment (MaximAlien) [reply MapboxNavigation/MGLStyle.swift]: Eventually I wasn't able to find anything for versions 4 and earlier, so added versions range check.

File: MapboxNavigation/MGLStyle.swift [hunk 4]
```diff
@@ -25,59 +31,45 @@ extension MGLStyle {
...
-     We only have one version of Navigation Preview style in China, so if you switch your endpoint to .cn, it will return the default day style.
-     */
-    public class func navigationPreviewDayStyleURL(version: Int) -> URL {
+    public class func navigationDayStyleURL(version: Int) -> URL {
         if MGLAccountManager.hasChinaBaseURL {
             return mapboxChinaDayStyleURL
         }
-        return URL(string:"mapbox://styles/mapbox/navigation-guidance-day-v\(version)")!
+        
+        if (1...4).contains(version) {
```
Comment (1ec5) [inline MapboxNavigation/MGLStyle.swift:51]: Cool. I think you can also use the `~=` operator here, but either way is more convenient than having to `&&` two conditions and risking an off-by-one error.

**Pull request merged**